racer.app.js

```diff
@@ -25,7 +31,7 @@ let pos2 = 150;
              }
```

Comment:
Watch out for indentation issues! If statements should line up with their closing brackets.

```
if(first_condition){
    // Code inside first if is indented one level
    if(second_condition) {
        // code inside nested if is indented another level
    } // Closing brace lines up with inner if
} // Closing brace lines up with outer if
```